README.md

```diff
@@ -79,7 +79,7 @@ Further information can be found in [Records Documentation](/docs/records.md).
 We're ready to do a dry-run with our new setup to see what changes it would make. Since we're pretending here we'll act like there are no existing records for `example.com.` in our accounts on either provider.
 
 ```
-$ octodns-sync --config-file=./config/production.yaml
+$ octodns-sync --config-file=./env/production.yaml
```

Comment:
`env` would be a common case for how to break these down, but not the only one. E.g. we might separate the public important zones, from the very static and seldom touched vanity redirects like gtihub.com -> github.com, and separate the constantly changing internal zones into yet another config. All of these would be "production", but there'd be (in this case) 3 different setups.

Reply:
@ross , maybe there is another name that will actually describe the content? `` config`` matches everything. Maybe ```inventory```?

Reply:
I think I'd like to stick to `config/` here since it's "config" for octoDNS itself. Inventory doesn't really seem to line up to me and I've not come up with anything else that seems to encompass all the cases.